client_options.go

```diff
@@ -268,6 +284,7 @@ func defaultClientOptions() *clientOptions {
 		gracefulShutdownPeriod:      30 * time.Second,
 		keepAlive:                   60 * time.Second,
 		credentialFetchTimeout:      10 * time.Second,
+		connectRetryPolicy:          connectRetryPolicy{interval: 10 * time.Second},
```

Comment:
can we only have retry policy with fixed retry intervals?

Reply:
Paho only supports fixed retry intervals as of now.